packages/@datapull/atlassian-origin/index.js

```diff
@@ -7,9 +7,10 @@ class AtlassianOrigin {
     this.organisation = config.organisation;
     this.username = config.username;
     this.password = config.password;
-    this.batchSize = Number(config.batchSize || 1000);
+    this.batchSize = Number(config.batchSize || 50);
     this.service = config.service;
-    if (this.service === 'jira' && this.batchSize > 1000) {
+    this.includeInactiveUsers = config.includeInactiveUsers || false;
+    if (this.service === 'jira' && this.batchSize > 50) {
```

Comment:
The underlying error still reports a 1K limit per call. IS the limit of the API endpoint 1K or did the default just change to 50? If the latter is the case, the if should still check wether the batchsize did not exceed 1K

Reply:
Changed! Maximum number of items per page for this endpoint is 50: 
https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-rest-api-3-group-member-get